docs/advanced.rst

```diff
@@ -758,6 +758,35 @@ Please refer to the supplemental example for details.
     length queries (``__len__``), iterators (``__iter__``), the slicing
     protocol and other kinds of useful operations.
 
+C++11 chrono datatypes
+======================
+When including the additional header file :file:`pybind11/chrono.h` conversions from c++11 chrono datatypes
```

Comment:
Another minor thing: `c++11` and `c++` should be capitalized to be consistent with the rest of the documentation (I found 4 lower-cased ones in all; this is the first one).